crates/sncast/src/response/ui.rs

```diff
@@ -0,0 +1,197 @@
+use foundry_ui::{Message, OutputFormat, UI as BaseUI};
+use serde::Serialize;
+use serde_json::Value;
+
+#[derive(Default)]
+pub struct UI {
+    ui: BaseUI,
+}
+
+struct MessageWrapper<T: Message> {
+    command: String,
+    message: T,
+}
+struct ErrorWrapper<T: Message> {
+    command: String,
+    error: T,
+}
+struct NotificationWrapper<T: Message> {
+    notification: T,
+}
+struct WarningWrapper<T: Message> {
+    warning: T,
+}
```

Comment:
Wdyt - would it make sense to add `command` to notification and wrapper as well?